### PR TITLE
TDR-3452 - Send the TRE v2 messages to the event bus SNS topic

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala [hunk 1]
```diff
@@ -370,7 +370,7 @@ object EventMessages {
   private def generateSnsExportMessageBody(bucketName: String,
                                            consignmentRef: String,
                                            producer: Producer): SnsMessageDetails = {
-    val topicArn = eventConfig("sns.topic.transform_engine_v2_in")
+    val topicArn = eventConfig("sns.topic.da_event_bus_arn")
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala:373]: You will need to update the `eventConfig` val in `Messages.scala` (line 36) to replace the `sns.topic.transform_engine_v2_in` entry with `da_event_bus_arn` entry. That should fix the unit tests as well, which are failing.

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of minor comments. Thanks

File: src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala (hunk 1 above)
Comment (vimleshtna) [reply src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala:373]: OK thanks

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -36,7 +36,7 @@ sqs {
 sns {
   endpoint = "http://localhost:9005"
   topic {
-    transform_engine_v2_in = "dHJlLWluLXRvcGljLWFybg==" # Base 64 decodes to this value tre-in-topic-arn
+    da_event_bus_arn = "da-eventbus-topic-arn"
```
Comment (TomJKing) [inline src/test/resources/application.conf:39]: This should be base 64 encoded. Sorry didn't state that.

Comment (vimleshtna) [reply src/test/resources/application.conf]: Oh sorry. I didn't realise too, thanks.

**Pull request merged**